Sources/Apollo/InMemoryNormalizedCache.swift

```diff
@@ -1,21 +1,37 @@
+import Dispatch
```

Comment:
I don't think you need to import this explicitly - it should be part of `Foundation`